README.md

```diff
@@ -51,6 +51,22 @@ Then, add the [runtime](docs/runtime.md) into your entry file (typically main en
 require('offline-plugin/runtime').install();
 ```
 
+... and for Typescript users:
```

Comment:
I think this should be a link to `docs/typescript.md`, e.g. "For TypeScript users see here"